src/modules/formBuilder/components/FormBuilder.tsx

```diff
@@ -171,7 +182,12 @@ const FormBuilder: React.FC<FormBuilderProps> = ({
               </Box>
             </Paper>
           </Box>
-          <SaveSlide in={isDirty} direction='up' loading={saveLoading}>
+          <SaveSlide
+            in={isDirty}
+            direction='up'
+            loading={saveLoading}
+            unmountOnExit
```

Comment:
`unmountOnExit` prevents the SaveSlide from adding extra padding to the bottom of the page when it isn't active. I wasn't sure if this prop should be added to the SaveSlide component itself, but decided not to mess with it since it's used in many places and would require more qa.